requirements/dev.txt

```diff
@@ -62,6 +62,11 @@ flask-sqlalchemy==3.1.1
     # via
     #   -r tests.txt
     #   -r typing.txt
+greenlet==3.0.3
```

Comment:
@davidism I'm surprised this wasn't previously in here? It's not like we changed our `sqlalchemy` dep at all...

Reply:
No idea. Perhaps sqlalchemy didn't depend on it by accident in a release?